src/Tests/Entities/ProjectTest.php

```diff
@@ -6,59 +6,54 @@
 use DateTime;
 use ProjectManagementApi\Entities\Project;
 use PHPUnit\Framework\TestCase;
-require '../../../vendor/autoload.php';
+
+require 'vendor/autoload.php';
```

Comment:
add an instruction in your README for how to run the unit tests (and particularly where to run them from)

Reply:
also this line should be at the top of the file i believe

Reply:
Done